### ISSUE TITLE
message in queue for rabbitmw

### ISSUE THREAD

**Issue by samratBasra** [akrzos/collectd-rabbitmq-monitoring#5]
@akrzos first of all thanks for writing this plugin. it works like charm. but i have one question that how can i see/monitor message in queue for rabbitmq.

**Comment by akrzos** [comment]
> @akrzos first of all thanks for writing this plugin. it works like charm. but i have one question that how can i see/monitor message in queue for rabbitmq.

@samratBasra can you provide more detail for your question?

**Comment by samratbe** [comment]
@akrzos yes. i have installed grafana for rabbitmq monitoring with this plugin. i am getting data for rabbitmq but i dont know how to see the message in queue for rabbitmq.

**Comment by tomaszwostal** [comment]
Hi, I have the same problem. Everything works great except msg_count metric. It shows 0 for each queue. If you need any additional information please let me know.

**Comment by akrzos** [comment]
@tomaszwostal Try adjusting which vhost your collecting that data from.  I am pretty sure the vhost configured in this plugin is set to '/' - https://github.com/akrzos/collectd-rabbitmq-monitoring/blob/master/collectd_rabbitmq_monitoring/__init__.py#L125

**Comment by tomaszwostal** [comment]
@akrzos I'm collecting data from '/'. I'm able to graph all metric except msg_count. When I use curl to get number of messages in queue I'm able to get the proper result.
`curl -s -u username:password http://hostname:15672/api/queues/%2F/my_queue | jq .messages
43
` 
To get proper results I had to change this piecie of code:
```
# Configurable per-queue message counts
    for queue_name in queues_to_count:
        messages_detail = None
        try:
            messages_detail = cl.get_queue('/', queue_name)['messages']
        except HTTPError as err:
            collectd.error(
                'Error Opening Queue [{}] details: {}'
                .format(queue_name, err))
        if messages_detail is None:
            count = 0
        else:
            count = messages_detail
```

**Comment by akrzos** [comment]
Got it pyrabbit might have changed since requirements.txt doesn't lock the pyrabbit version down.
